packages/zk-transfer-web/src/helpers/account.ts

```diff
@@ -0,0 +1,6 @@
+export const shortenAddress = (address: string) => {
```

Comment:
No need to shorten the address here, although it's a cool idea.

I think it's more confusing for experienced Ethereum users to see the `...` in the middle. You can widen the `div` to display the full address if necessary.

Reply:
Okay. I'll done

Reply:
Can you also delete (and commit/push) the definition of this `shortenAddress` function since it's not needed anymore? Thanks

Reply:
Great!

Reply:
Pushing now!